add_gem_package.sh

```diff
@@ -174,13 +176,20 @@ if [[ ! -e $TEMPLATE ]] ; then
 	exit 1
 fi
 
-generate_gem_package
 if [[ $UPDATE == true ]] ; then
-	VERSION=$(rpmspec --srpm -q --queryformat="%{version}-%{release}" --undefine=dist $PACKAGE_DIR/$SPEC_FILE)
-	git commit -m "Bump $PACKAGE_NAME to $VERSION"
+  EXISTING_VERSION=$(rpmspec --srpm -q --queryformat="%{version}" $PACKAGE_DIR/$SPEC_FILE)
+  NEW_VERSION=$(curl -s https://rubygems.org/api/v1/gems/${GEM_NAME}.json | jq -r .version)
+  if [[ $REWRITE_ON_SAME_VERSION == true ]] || [[ $NEW_VERSION != $EXISTING_VERSION ]]; then
+    generate_gem_package
+    VERSION=$(rpmspec --srpm -q --queryformat="%{version}-%{release}" --undefine=dist $PACKAGE_DIR/$SPEC_FILE)
+    git commit -m "Bump $PACKAGE_NAME to $VERSION"
+  else
+    echo "$PACKAGE_NAME is already at version $VERSION"
+  fi
 else
-	add_to_manifest
-	add_to_tito_props
-	add_gem_to_comps
-	git commit -m "Add $PACKAGE_NAME package"
+  generate_gem_package
```

Comment:
The rest of the file uses tabs for indenting. Consistency would be lovely :)

Reply:
You want me to convert the whole file to spaces? ;-)